crates/starknet-server/tests/common/data.rs

```diff
@@ -0,0 +1,24 @@
+use starknet_in_rust::core::contract_address::compute_casm_class_hash;
+use starknet_in_rust::CasmContractClass;
+use starknet_rs_core::types::contract::SierraClass;
+use starknet_rs_core::types::{FieldElement, FlattenedSierraClass};
+use starknet_types::felt::Felt;
+
+pub fn get_events_contract_in_sierra_and_compiled_class_hash()
```

Comment:
Can be moved to utils.rs
Imo name `data.rs `is a bit confusing

Reply:
moved to utils.rs